requirements.txt

```diff
@@ -1,11 +1,10 @@
 # The "base" version of the Mkdocs project.
-# Note: if you update this, also update `install_requires` in setup.py
 # https://github.com/mkdocs/mkdocs
-mkdocs==1.1.2
+mkdocs>=1.2.1
```

Comment:
Note: This might actually break the serve/preview/live reloading. See https://github.com/mkdocs/mkdocs/issues/2457.

Reply:
Now fixed in https://github.com/mkdocs/mkdocs/releases/tag/1.2.2